<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AmazonConfig</class>
 <widget class="QDialog" name="AmazonConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Amazon configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horlayServer">
     <item>
      <widget class="QLabel" name="labelServer">
       <property name="text">
        <string>Server:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineServer"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horlayAKID">
     <item>
      <widget class="QLabel" name="labelAKID">
       <property name="text">
        <string>Access Key ID:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineAKID"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horlaySAK">
     <item>
      <widget class="QLabel" name="labelSAK">
       <property name="text">
        <string>Secret Access Key:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineSAK"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupShaType">
     <property name="title">
      <string>Sign algorithm</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="radioSha1">
        <property name="text">
         <string>Sha1</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioSha256">
        <property name="text">
         <string>Sha256</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horlayOkCancel">
     <item>
      <widget class="QPushButton" name="pushLoad">
       <property name="text">
        <string>Load from XML</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushSave">
       <property name="text">
        <string>Save to XML</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushOK">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineServer</tabstop>
  <tabstop>lineAKID</tabstop>
  <tabstop>lineSAK</tabstop>
  <tabstop>radioSha1</tabstop>
  <tabstop>radioSha256</tabstop>
  <tabstop>pushLoad</tabstop>
  <tabstop>pushSave</tabstop>
  <tabstop>pushOK</tabstop>
  <tabstop>pushCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushOK</sender>
   <signal>clicked()</signal>
   <receiver>AmazonConfig</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>271</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushCancel</sender>
   <signal>clicked()</signal>
   <receiver>AmazonConfig</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
